en_US.json

```diff
@@ -1267,6 +1273,8 @@
 	"hint.variable.$tcpinfo_delivery_rate": "acked data rate in bytes/s",
 	"hint.variable.$tcpinfo_min_rtt": "min RTT in us (microseconds)",
 	"hint.variable.$tcpinfo_rtt": "RTT in us (microseconds)",
+	"hint.variable.$transfer_time": "time spent on sending full response to system buf",
+	"hint.variable.$turn_around_time": "time to receiving 1st byte from origin",
```

Comment:
English correction:    "time to receive 1st byte from origin"

Reply:
Sorry, I misinterpreted.  I checked the Edge Logic documentation and believe it is better stated as  "time until receiving 1st byte from origin".